tests/test_full_table_interruption.py

```diff
@@ -200,8 +229,16 @@ def test_catalog(self):
         self.assertEqual(CAUGHT_MESSAGES[7].value['bookmarks']['public-COW'].get('lsn'), end_lsn)
         self.assertEqual(CAUGHT_MESSAGES[7].value['bookmarks']['public-COW'].get('version'), new_version)
 
+        assert CAUGHT_MESSAGES[8]['type'] == 'SCHEMA'
+
+        assert isinstance(CAUGHT_MESSAGES[9], singer.messages.StateMessage)
+        assert CAUGHT_MESSAGES[9].value['bookmarks']['public-COW'].get('xmin') is None
+        assert CAUGHT_MESSAGES[9].value['bookmarks']['public-COW'].get('lsn') == end_lsn
+        assert CAUGHT_MESSAGES[9].value['bookmarks']['public-COW'].get('version') == new_version
+
```

Comment:
@Samira-El I'm facing this issue and planning on contributing the fix. Is your feedback related to line 268 only? That is change it to `assertIsInstance(CAUGHT_MESSAGES[9], singer.messages.StateMessage)`

Reply:
@Samira-El Since @judahrand is unavailable to continue with this [PR](https://github.com/transferwise/pipelinewise-tap-postgres/issues/107#issuecomment-1331883206), I'm interested in contributing the solution once you highlight the work that is missing ^ I've re-opened the PR [here](https://github.com/transferwise/pipelinewise-tap-postgres/pull/218), with master branch merged and resolved conflicts